style/index.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title>Display a map</title>
+    <meta
+      name="viewport"
+      content="initial-scale=1,maximum-scale=1,user-scalable=no"
+    />
+    <script src="https://unpkg.com/maplibre-gl@1.14.0/dist/maplibre-gl.js"></script>
+    <link
+      href="https://unpkg.com/maplibre-gl@1.14.0/dist/maplibre-gl.css"
+      rel="stylesheet"
+    />
+    <style>
+      body {
+        margin: 0;
+        padding: 0;
+      }
+
+      #map {
+        position: absolute;
+        top: 0;
+        bottom: 0;
+        width: 100%;
+      }
+    </style>
+
+    <!-- This file needs to be modified with your MapTiler Cloud key -->
+    <script type="text/javascript" src="key.js"></script>
+
+    <script type="text/javascript" src="constants/color.js"></script>
+    <script type="text/javascript" src="constants/line.js"></script>
+
+    <script type="text/javascript" src="layer/background.js"></script>
+    <script type="text/javascript" src="layer/water.js"></script>
+    <script type="text/javascript" src="layer/road_casing.js"></script>
+    <script type="text/javascript" src="layer/road.js"></script>
+    <script type="text/javascript" src="layer/bridge.js"></script>
+    <script type="text/javascript" src="layer/tunnel.js"></script>
+    <script type="text/javascript" src="layer/highway_shield.js"></script>
+    <script type="text/javascript" src="layer/highway_exit.js"></script>
```

Comment:
`layer/highway_exit.js` is missing

Reply:
Ah, that's a leftover.  It seems there's no highway exit support out of the box in OpenMapTiles, so I'm taking it out.